src/player.cpp

```diff
@@ -7756,6 +7756,15 @@ bool player::wield( item& target )
     if( target.is_null() ) {
         return true;
     }
+
+    // Query whether to draw an item from a holster when attempting to wield the holster
+    if( target.get_use( "holster" ) && ( target.contents.size() > 0 ) ) {
+        if( query_yn( string_format( _( "Draw %s from %s?" ), target.get_contained().tname().c_str(), target.get_contained().tname().c_str()) ) ) {
```

Comment:
Second string is wrong, "Draw Beretta M9 from Beretta M9?"

Reply:
BTW, this line itself is way too long, please target on 100 symbols max in one line (see astyle rules for details).